src/controllers/queries.controllers.ts

```diff
@@ -0,0 +1,69 @@
+import { Request, Response } from "express";
+import Queries, { IQueries } from "../models/queries.model";
+
+// get all Queries
+export const getQueries = async (
+  req: Request,
+  res: Response
+): Promise<void> => {
+  try {
+    const queries: IQueries[] = await Queries.find({});
+    res.status(200).json({ data: queries });
+  } catch (error) {
+    res.status(500).json({ message: (error as Error).message });
+  }
+};
+
+// Create a new Query
+export const createQuery = async (
+  req: Request,
+  res: Response
+): Promise<void> => {
+  try {
+    const newQuery: IQueries = new Queries(req.body);
+    const savedQuery: IQueries = await newQuery.save();
+    res
+      .status(201)
+      .json({ message: "Message Sent successfully", data: savedQuery });
+  } catch (error) {
+    res.status(500).json({ message: (error as Error).message });
+  }
```

Comment:
Rather than writing try catch everytime, can it be  a good way to do that aside and use it everywhere,

Reply:
okay  cool , let me  found out the way i can fix that